### ISSUE TITLE
Automation for backups / Continual Backups

### ISSUE THREAD

**Issue by X0RA** [Tyrrrz/DiscordChatExporter#32]
Hey Tyrrrz,

I'm wondering if automated backups are apart of the scope for this project? Being able to have it backup every day at a certain time would be great. 

This is for deletions that occur in the server, having a history would be very beneficial and the manual backups would catch some deleted messages but no all of them. 

Another way i could see this implemented is that it reads messages as they are posted and archives them? This way it would ensure that all messages that are deleted are archived.

I really appreciate this project, there aren't many around

**Comment by Tyrrrz** [comment]
Hi.

With the command line interface you can probably set up a recurring job to export channels you're interested in. Other than that, there is no built-in feature that supports this.

**Comment by aipine** [comment]
Ability to do this is desperately needed. I need a rolling log of my server and unfortunately, I do not possess the knowledge to set it up via the command line.

**Comment by zarzwour** [comment]
i also want this

**Comment by TuckMeIntoBread** [comment]
This, or at least some capability to mass-export would be nice.

**Comment by gregorybloom** [comment]
Got it.  I think I've got a good grasp of using the CLI now (via Python).

I'm looking for a way to get a list of the channel id's for a server, but... eh!

**Comment by Tyrrrz** [comment]
You can't currently get that with DCE

**Comment by gregorybloom** [comment]
Yes.  I more meant off the Discord Page.  I can grab the channel list for private messages, but the server page elements don't contain their channel list at all.  It's either obscured in JS or held somewhere else.

**Comment by gregorybloom** [comment]
Could be.  Right now you have to sign into the discord web app anyway (for token access), but right now I have extra steps to:
Fetch the List of Private Message Channel ID's and Names, Fetch the List of Server ID's, and now I'm trying to Fetch the List of Channel ID's and Names for a given Server ID.  

Having at least these three commands in the CLI interface would allow people to automate the fetching of some (or all) of their logs...  But it depends how technically difficult those steps would be to do.  The PM ID's and Channel ID's for a Server are likely the most valuable pieces of information.

**Comment by ThatGmodZServer** [comment]
If I could have a way to backup all contacts not servers, I would love that feature instead of having to individually export each one.
